l10n_it_fatturapa_sale/__manifest__.py

```diff
@@ -16,6 +16,7 @@
         "l10n_it_fatturapa",
         "sale_management",
     ],
+    "conflicts": ["sale_order_action_invoice_create_hook"],
```

Comment:
Non dovrebbe essere `excludes`?

Reply:
Avevo copiato dall'unico caso in cui c'era `conflicts` :'(
Grazie!